src/message.ts

```diff
@@ -403,6 +403,37 @@ export class Message implements Sayable {
     return fromId === userId
   }
 
+  /**
+   * Get the mentioned contact which the message is mentioned for.
+   * @returns {Contact[]} return the contactList which the message is mentioned for
+   */
+  public mention(): Contact[] {
+    let contactList: Contact[] = []
+    const room = this.room()
+    if (!room) return contactList
+    if (this.type() !== MsgType.TEXT) return contactList
+
+    const foundList = this.content().match(/@\S+ ?/g)
```

Comment:
yes, the foundeList will contains all the **shortest** string following by a `@` character.

If ones' wechat name is `lijiarui@orange`, it will return `orange`, instead of `lijiarui@orange`.

Or, we should permutation and combination all the possibility.....

What do you think about it?